sdk/eventgrid/eventgrid/src/generated/models/index.ts

```diff
@@ -3039,6 +3039,273 @@ export interface WebAppServicePlanUpdatedEventDataSku {
+ */
+export type ACSChatMessageDeletedEventData = ACSChatMessageEventBaseProperties & {
+  /**
+   * The time at which the message was deleted
+   */
+  deleteTime?: Date;
+};
+
+/**
+ * Schema of common properties of all chat thread events
+ */
+export type ACSChatThreadEventBaseProperties = ACSChatEventBaseProperties & {
+  /**
+   * The original creation time of the thread
+   */
+  createTime?: Date;
+  /**
+   * The version of the thread
+   */
+  version?: number;
+};
+
+/**
+ * Schema of the Data property of an EventGridEvent for an Microsoft.Communication.ChatThreadCreatedWithUser event.
+ */
+export type ACSChatThreadCreatedWithUserEventData = ACSChatThreadEventBaseProperties & {
+  /**
+   * The MRI of the creator of the thread
+   */
+  createdBy?: string;
+  /**
+   * The thread properties
+   */
+  properties?: { [propertyName: string]: any };
+  /**
+   * The list of properties of users who are part of the thread
+   */
+  members?: ACSChatThreadMemberProperties[];
+};
+
+/**
+ * Schema of the chat thread member
+ */
+export interface ACSChatThreadMemberProperties {
+  /**
+   * The name of the user
+   */
+  displayName?: string;
+  /**
+   * The MRI of the user
+   */
+  memberId?: string;
+}
+
+/**
+ * Schema of the Data property of an EventGridEvent for an Microsoft.Communication.ChatThreadWithUserDeleted event.
+ */
+export type ACSChatThreadWithUserDeletedEventData = ACSChatThreadEventBaseProperties & {
+  /**
+   * The MRI of the user who deleted the thread
+   */
+  deletedBy?: string;
+  /**
+   * The deletion time of the thread
+   */
+  deleteTime?: Date;
+};
+
+/**
+ * Schema of the Data property of an EventGridEvent for an Microsoft.Communication.ChatThreadPropertiesUpdatedPerUser event.
+ */
+export type ACSChatThreadPropertiesUpdatedPerUserEventData = ACSChatThreadEventBaseProperties & {
+  /**
+   * The MRI of the user who updated the thread properties
+   */
+  editedBy?: string;
+  /**
+   * The time at which the properties of the thread were updated
+   */
+  editTime?: Date;
+  /**
+   * The updated thread properties
+   */
+  properties?: { [propertyName: string]: any };
+};
+
+/**
+ * Schema of the Data property of an EventGridEvent for an Microsoft.Communication.ChatMemberAddedToThreadWithUser event.
+ */
+export type ACSChatMemberAddedToThreadWithUserEventData = ACSChatThreadEventBaseProperties & {
+  /**
+   * The time at which the user was added to the thread
+   */
+  time?: Date;
+  /**
+   * The MRI of the user who added the user
+   */
+  addedBy?: string;
+  /**
+   * The details of the user who was added
+   */
+  memberAdded?: ACSChatThreadMemberProperties;
+};
+
+/**
+ * Schema of the Data property of an EventGridEvent for an Microsoft.Communication.ChatMemberRemovedFromThreadWithUser event.
+ */
+export type ACSChatMemberRemovedFromThreadWithUserEventData = ACSChatThreadEventBaseProperties & {
+  /**
+   * The time at which the user was removed to the thread
+   */
+  time?: Date;
```

Comment:
I wonder if it would make sense to standardize time properties with `timestamp`. For example time > timestamp or editTime > editTimestamp

Reply:
Making a note of this for the larger pass that will happen as part of https://github.com/Azure/azure-sdk-for-js/issues/10992